web/src/settings_org.ts

```diff
@@ -603,7 +603,8 @@ export function discard_stream_property_element_changes(
         sub,
     );
     switch (property_name) {
-        case "can_remove_subscribers_group": {
+        case "can_remove_subscribers_group":
+        case "can_administer_channel_group": {
```

Comment:
Similarly, this one can probably be nicely refactored to check if it's in the group-permissions set before the `switch` statement, and handle that block being careful about the line after the `switch` statement, to avoid needing an explicit change here whenever we add a new setting.

Reply:
Fixed in #32558 (2nd commit)